sw/airborne/arch/chibios/mcu_arch.h

```diff
@@ -44,4 +44,40 @@ extern void mcu_arch_init(void);
 extern bool recovering_from_hard_fault;
 #endif
 
+#include <ch.h>
+
+static inline void mcu_deep_sleep(void)
```

Comment:
it is required by the SDLOG system of Chimera and Apogee when closing the files on power down detection